crates/ark/src/modules.rs

```diff
@@ -292,6 +292,21 @@ fn r_poke_option_ark_testing() {
     }
 }
 
+#[harp::register]
+pub extern "C-unwind" fn ark_log_trace(msg: SEXP) -> harp::error::Result<SEXP> {
+    Ok(harp::utils::harp_log_trace(msg))
+}
```

Comment:
I've needed these before from the ark modules side

When you're in ark, you can't access the R level `log_trace()` and friends that exist in the R level harp modules